src/main/java/org/broadinstitute/hellbender/utils/test/SamAssertionUtils.java

```diff
@@ -363,6 +371,44 @@ public static void assertEqualBamFiles(final File resultFile, final File expecte
         }
     }
 
+    // validate that this file has CRAM contents (no matter what the extension)
+    protected static boolean isCRAMFile(final File putativeCRAMFile, final File referenceFile) {
```

Comment:
Move to `ReadUtils` and make public. This should not be buried in `SamAssertionUtils`, since it returns a boolean rather than performing an assertion.